### ISSUE TITLE
download multiple LRs

### ISSUE THREAD

**Issue by MariaGiagkou** [MiltosD/ELRC-Client#3]
my command: download 281 249 280 282 283 278 279

i got most of them ok, but 
[2018-12-06 14:52:22,578] ELRC-SHARE::ERROR  Dataset for resource 278 was not found

[2018-12-06 14:52:22,613] ELRC-SHARE::ERROR  Dataset for resource 279 was not found
Although i can see these ids in the list of LRs in manage resources

**Comment by MiltosD** [comment]
Did you actually upload datasets for these resources (278, 279)? If not, then I can just change the message.

**Comment by MariaGiagkou** [comment]
No, i did not. However, it is difficult to remember the correspondence of ids to LR name and to file name. I mean for example that in the first step i created an LR from my files with filename resource-1.xml. Then this was assigned an id that i am not able to figure out

**Comment by MiltosD** [comment]
I have added a **list** functionality that returns resource ids, name, and status, plus when you create a resource you now also get the id of the new resource. I will leave this open until is retested
